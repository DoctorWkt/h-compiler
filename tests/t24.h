void main()
{
  int o_ll=66;
  int m=0;
  int x= (o_ll - m) / 2;
  int j;
  int l;
  printn(x);
  for (j = l; j < x; j++) printn(j);
  putchar('=');
  putchar('\n');
  for (j = l; j < (o_ll - m) / 2; j++) printn(j);
}
