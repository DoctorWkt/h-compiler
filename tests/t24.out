000041
000000
000001
000002
000003
000004
000005
000006
000007
000010
000011
000012
000013
000014
000015
000016
000017
000020
000021
000022
000023
000024
000025
000026
000027
000030
000031
000032
000033
000034
000035
000036
000037
000040
=
000000
000001
000002
000003
000004
000005
000006
000007
000010
000011
000012
000013
000014
000015
000016
000017
000020
000021
000022
000023
000024
000025
000026
000027
000030
000031
000032
000033
000034
000035
000036
000037
000040
